geth/jail/jail_test.go

```diff
@@ -115,6 +128,46 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
+	require := s.Require()
+
+	defer func() {
+		if err := recover(); err != nil {
+			require.NotNil(err, "Panic occured")
+		}
+	}()
+
+	require.NotNil(s.jail)
+
+	s.StartTestNode(params.RopstenNetworkID)
+	defer s.StopTestNode()
+
+	// load Status JS and add test command to it
+	s.jail.BaseJS(baseStatusJSCode)
```

Comment:
Honestly, I see no problems in these places.
Anyway, don't worry about this, moved the question into #247